Cargo.toml

```diff
@@ -14,6 +14,9 @@ categories = ["asynchronous", "concurrency"]
 [dependencies]
 event-listener = "2.5.1"
 
+[target.'cfg(target_arch = "wasm32")'.dependencies]
+instant = { version = "0.1.9" }
```

Comment:
I think this has link issue like: https://github.com/Amanieu/parking_lot/issues/269

Reply:
Hmm, then maybe we should just remove the use of `std::time::Instant` entirely when targeting WASM? That seems reasonable, because, like they said in the parking lot issue, WASM only has one thread so fair locking shouldn't be needed.